Hookies/Game/Engine/GameEngine.swift

```diff
@@ -27,10 +27,12 @@ class GameEngine {
     private var userConnectionSystem: UserConnectionSystem?
     private var startSystem = StartSystem()
     private var endSystem: EndSystem?
+    private var botSystem: BotSystem?
 
     // MARK: - Entity
 
     private var currentPlayer: PlayerEntity?
+    private var localPlayers = [String: PlayerEntity]()
```

Comment:
Should remove this. We shouldn't be using `player id` inside the game engine.

Reply:
Maybe can change to an array instead.

Reply:
Should be able to have it as an array of playerEntity